homeassistant/components/xiaomi_miio/alarm_control_panel.py

```diff
@@ -0,0 +1,201 @@
+"""
+Support for Xiomi Gateway alarm control panels.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/
```

Comment:
Is this boilerplate still in the integration templates? I thought that this is not anymore wanted..

Reply:
You are right, I changed it in the latest commit